src/components/WwwFrame/TheHeader.vue

```diff
@@ -9,6 +16,57 @@
 					</router-link>
 				</div>
 			</template>
+			<template v-else-if="corporate">
+				<div class="header-row row">
+					<div class="header-logo ">
+						<kiva-logo class="icon" />
+						<span class="show-for-sr">Kiva</span>
+						<div class="header-corporate-logo" v-if="!!this.$slots.corporateLogo">
+							<slot name="corporateLogo"></slot>
+						</div>
+					</div>
+					<div class="flexible-center-area"></div>
+					<router-link
+						v-show="showBasket"
+						to="/basket"
+						class="header-button show-for-large"
+						v-kv-track-event="['TopNav','click-Basket']"
+					>
+						<span>
+							<span class="amount">{{ basketCount }}</span>
+							Basket
+						</span>
+					</router-link>
+					<router-link
+						v-show="isVisitor"
+						:to="loginUrl"
+						class="header-button"
+						:event="showPopupLogin ? '' : 'click'"
+						@click.native="auth0Login"
+						v-kv-track-event="[
+							['TopNav','click-Sign-in'],
+							['TopNav','EXP-GROW-282-Oct2020',redirectToLoginExperimentVersion]
+						]"
+					>
+						<span>Sign in</span>
+					</router-link>
+					<router-link
+						v-show="!isVisitor"
+						:id="myKivaMenuId"
+						to="/portfolio"
```

Comment:
Same here, no navigating away from this page. If the hash thing works we can do similar here. if not we can just prevent navigation.